src/main/public/js/double-click-preventer.js

```diff
@@ -0,0 +1,18 @@
+(function () {
+  document.addEventListener("DOMContentLoaded", function (event) {
```

Comment:
although its not enforced for the frontend javascript our codestyle is to use `standardjs`
which means always use single quotes and no semi-colons

Reply:
https://app.codacy.com happen to be moaning about double quotes and semi colon hence I decided to do it. Their argument on double quote felt quite reasonable to me as well.

Reply:
ignore codacy when it violates our coding style please